soml/db-soml/wev-model.yaml

```diff
@@ -124,6 +124,40 @@ objects:
     prefix: "Review/Normalized/"
     inherits: Rating
     descr: "Evaluation of a Claim on a normalized scale: FALSE, OTHER, TRUE"
+
+  # Enrichments
+  Mention:
+    type: [wev:Mention]
+    prefix: "Mention/"
+    descr: "A specific instance of a concept being mention in the text of a document"
+    props:
+      description: {descr: "The text that mentions a concept", min: 1}
+      mentionedIn: {desc: "The object that contains the concept mention.", min: 1}
+      concept: {descr: "The concept that is mentioned", rdfprop: wev:hasConcept, range: DefinedTerm, min: 1}
+      confidence: {descr: "Confidence in the accuracy of the concept.", rdfprop: wev:confidence, range: decimal}
+      relevance: {descr: "Relevance of the concept to the document.", rdfprop: wev:relevance, range:decimal}
+      startOffset: {descr: "The beginning offset of the mention in the document text.", range: integer}
+      endOffset: {descr: "The end offset of the mention in the document text.", range: integer}
+
```

Comment:
Do we need one more level of abstraction to hold metadata about the system that produced the enrichment? Should it be added to the mention object directly? Do we not even need it? Do we want to keep it in the graph but not expose it to GQL?

Reply:
I'm sure we need the data about the system that produced the enrichment. It could be a separate object and only link it from the mention, right? We can have such objects for each system which annotates. I need more context regarding the final objective of the service in order to answer about exposing this data.